plugins/meta/aos-firewall/aosfirewall_internal_test.go

```diff
@@ -223,7 +224,7 @@ var _ = Describe("Aos Firewall", func() {
 			_, _, err = testutils.CmdAdd(targetNS.Path(), args.ContainerID, IFNAME, []byte(fullConf), func() (err error) {
 				return cmdAdd(args)
 			})
-			Expect(err).NotTo(HaveOccurred())
+			Expect(err).To(HaveOccurred())
```

Comment:
How did it work before?

Reply:
previous formatIptablesRequest didn't return an error, now it's return error